web/html/src/manager/content-management/list-filters/filter-form.tsx

```diff
@@ -63,162 +69,193 @@ const FilterForm = (props: Props) => {
             {t("Bear in mind that all the associated projects need to be rebuilt after a filter update")}
           </div>
         )}
-        <div className="row">
+        {filterBy === FilterBy.Type ? (
           <Text
+            key="filter_name"
             name="filter_name"
             label={t("Filter Name")}
             labelClass="col-md-3"
             divClass="col-md-6"
             required
             disabled={props.editing}
           />
-        </div>
-
-        <Select
-          name="type"
-          label={t("Filter Type")}
-          labelClass="col-md-3"
-          divClass="col-md-6"
-          required
-          disabled={props.editing}
-          options={getClmFiltersOptions()}
-          getOptionValue={filter => filter.key}
-          formatOptionLabel={filter => `${filter.entityType.text} (${filter.text})`}
-        />
-
-        {selectedFilterMatchers && (
-          <Select
-            name="matcher"
-            label={t("Matcher")}
+        ) : (
+          <Text
+            key="labelPrefix"
+            name="labelPrefix"
+            label={t("Prefix")}
             labelClass="col-md-3"
             divClass="col-md-6"
             required
             disabled={props.editing}
-            options={selectedFilterMatchers}
-            getOptionValue={matcher => matcher.key}
-            getOptionLabel={matcher => matcher.text}
```

Comment:
To avoid confusion, we need to explain this new feature very well to users, considering they might not read all the updates in the docs in advance. For this, a help text could be useful here. It could say something like:

> The prefix will be prepended to every individual filter created by the template

Reply:
Good idea, completely agreed.